packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -31,7 +31,7 @@
     "@salesforce/salesforcedx-utils-vscode": "62.3.1",
     "ajv": "6.12.6",
     "applicationinsights": "1.0.7",
-    "jest-editor-support": "^30.1.0",
```

Comment:
I pinned the version since 30.3.1 is the latest minor of v30. And the latest major is 32.0.1-beta so there will not be any update with v30.